nixos/modules/programs/wayland/niri.nix

```
@@ -12,12 +12,21 @@ in
     enable = lib.mkEnableOption "Niri, a scrollable-tiling Wayland compositor";
 
     package = lib.mkPackageOption pkgs "niri" { };
+
+    xwaylandSatellite = {
+      enable = lib.mkEnableOption "Enable xwayland-satellite for Niri";
```

this will be generated as `Whether to enable Enable...`
```suggestion
      enable = lib.mkEnableOption "xwayland-satellite for Niri";
```